### PR TITLE
feat: Endpoint to get QR json offer

### PR DESCRIPTION
**Pull request by x1m3**
Endpoint to get a json  to give instructions to the mobile app.

Closes https://polygonid.atlassian.net/browse/ID-149

### CODE REVIEW

File: internal/api/server.go [hunk 1]
```diff
@@ -244,6 +247,67 @@ func (s *Server) GetClaims(ctx context.Context, request GetClaimsRequestObject)
...
+				},
+			},
+			Url: fmt.Sprintf("%s/v1/agent", strings.TrimSuffix(hostURL, "/")),
+		},
+		From: claim.Issuer,
+		Id:   id.String(),
+		Thid: id.String(),
+		To:   *claim.Identifier,
+		Typ:  "application/iden3comm-plain-json",
+		Type: "https://iden3-communication.io/credentials/1.0/offer",
```
Comment (javip97) [inline internal/api/server.go:307]:
```
packers.MediaTypePlainMessage
protocol.CredentialIssuanceResponseMessageType
```

File: api/api.yaml [hunk 2]
```diff
@@ -292,6 +292,31 @@ paths:
           $ref: '#/components/responses/400'
         '500':
           $ref: '#/components/responses/500'
+  /v1/{identifier}/claims/{id}/qrcode:
+    get:
+      summary: Get Claim QR code
+      operationId: GetQrCodeClaim
```
Comment (javip97) [inline api/api.yaml:298]: I would put `GetClaimQrCode` instead of `GetQrCodeClaim`

Comment (javip97) [reply api/api.yaml:298]: The same with the object response (`GetQrCodeClaimResponse`)

Comment (x1m3) [reply api/api.yaml]: Done

File: internal/api/server.go (hunk 1 above)
Comment (x1m3) [reply internal/api/server.go]: Done

Comment (javip97) [reply internal/api/server.go]: my bad, the type should be 

`Type: string(protocol.CredentialOfferMessageType)`

**Review by javip97 — APPROVED** [review]
Nice Job!!!

**Pull request merged**